src/pages/using-spark/guides/validation-messaging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+  title: Validation Messaging
+---
+import validationAnatomy from '../../../images/guides/validation-anatomy.png';
+import validationAnatomy2x from '../../../images/guides/validation-anatomy@2x.png';
+export const validationAnatomySet = `${validationAnatomy} 560w, ${validationAnatomy2x} 1120w`
+
+import {
+  SprkHeading,
+} from '@sparkdesignsystem/spark-react';
+
+<span id="validation-messaging-guide" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h1" additionalClasses="sprk-u-mbl sprk-u-Measure" isPageTitle>
+  Validation Messaging Guide
+</SprkHeading>
+
+<span id="main-takeaways" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Main Takeaways
+</SprkHeading>
+
+Giving validation feedback is an emotional process. Support users with this process by following these guidelines:
+
+- Explain what the Input expects before the user enters data. (Example: password rules, ranges, etc)
+- Give clear and helpful feedback when encountering a problem.
+- Use a combination of [placement](#placement), [timing](#timing), and [language](#language) to guide users through moments of uncertainty.
+
+<img
+  srcSet={validationAnatomySet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbh"
```

Comment:
Adding Measure here overides the default max-width: 100% that is applied to all images and forces this image to not be able to scale to a smaller size on mobile. It makes half of the image get cut off and makes a horizontal scrollbar.
<img width="320" alt="Screen Shot 2021-03-18 at 5 25 43 PM" src="https://user-images.githubusercontent.com/308696/111699895-13065080-880f-11eb-8c15-68a5a5bb4834.png">

Reply:
![image](https://user-images.githubusercontent.com/4342363/111703400-f587b580-8813-11eb-87cb-b9ebd26ace01.png)
done!